### ISSUE TITLE
PROBLEM : FOLLOWING  !

### ISSUE THREAD

**Issue by topliftarm** [twintproject/twint#1127]
now i cant retrieve Following of users!

i use this in google colab : 
 
 c = twint.Config()
 c.Username = "myName"
 c.Store_csv = False
 c.Following = True
 c.Output = c.Username + "_following.txt"
 c.Hide_output = True
 twint.run.Following(c)

but i give just these -> 

CRITICAL:root:twint.feed:Follow:IndexError
CRITICAL:root:twint.get:User:'NoneType' object is not subscriptable

for many account checked and results is same

**Comment by topliftarm** [comment]
please HELP me

**Comment by dbohdan** [comment]
This affects me, too, though I use the command line interface.

```sh
> twint -u jack --following --user-full
CRITICAL:root:twint.feed:Follow:IndexError
```

**Comment by tryptofanik** [comment]
Same for me. I have checked and it seems that response is empty...

**Comment by Vix18** [comment]
I have the same issue! Is twint still being updated? Are there any other osint tools that can help with this job - i.e. getting a list of followings?

**Comment by dit7ya** [comment]
Yeah, same issue for me. I will watch this thread for any workarounds if anybody finds.

**Comment by jcsimmons1** [comment]
Anyone out there on this error for a user's followers/following? CRITICAL:root:twint.feed:Follow:IndexError Saw another thread where it had to do with  "That's because twitter killed the no js (mobile version) version of twitter on 15th December. Followers were scraped using that version." https://github.com/twintproject/twint/issues/409

**Comment by nate8020** [comment]
Also got the issue of `CRITICAL:root:twint.feed:Follow:IndexError`.

Anybody found a workaround?

Following.

**Comment by anhagape** [comment]
following

**Comment by anhagape** [comment]
anyone found a workaround.

**Comment by jcsimmons1** [comment]
I don't believe there is one Twitter killed the way Twint was using to pull followers. I use TweetBeaver for this kind of work now.

**Comment by DV777** [comment]
Please if anyone manage to solve this issue, add it here...

**Comment by youssefavx** [comment]
Does anyone have any alternatives to twint as @jcsimmons1 suggested? for tweets, I've found snscrape to be good. But for following, favorites, followers, etc. I'd love to find an alternative because snscrape does not support those.

**Comment by dt140120** [comment]
Same issues, just not changing
